lms/views/email.py

```diff
@@ -39,11 +39,11 @@ def __init__(self, request):
     )
     def unsubscribe(self):
         """Unsubscribe the email and tag combination encoded in token."""
-        self.request.find_service(EmailPreferencesService).unsubscribe(
-            self.request.identity.h_userid
-        )
+        self.request.find_service(
+            EmailPreferencesService
+        ).instructor_digest_unsubscribe(self.request.identity.h_userid)
         self.request.session.flash(
-            "You've been unsubscribed from email notifications.",
+            "You've been unsubscribed from instructor digest.",
```

Comment:
Be more precise here.

Reply:
I don't think we actually refer to these emails as "digests" anywhere in the email preferences UI or in the emails themselves, so instructors won't know what this means. Maybe:

> You've been unsubscribed from student annotation email notifications.

Screenshots of the user experience:

<img width="723" alt="Screenshot 2025-03-26 at 11 08 08" src="https://github.com/user-attachments/assets/52d693ca-f4d0-4fe8-9b6a-62cdd640c745" />
<img width="471" alt="Screenshot 2025-03-26 at 11 08 52" src="https://github.com/user-attachments/assets/cf0de7c2-b22a-4f57-9f43-9b5429af359c" />